Include/iterobject.h

```diff
@@ -17,6 +18,11 @@ PyAPI_FUNC(PyObject *) PySeqIter_New(PyObject *);
 
 PyAPI_FUNC(PyObject *) PyCallIter_New(PyObject *, PyObject *);
 
+
+#define PySeqAIter_Check(op) Py_IS_TYPE((op), &PySeqAIter_Type)
+
+PyAPI_FUNC(PyObject *) PySeqAIter_New(PyObject *);
```

Comment:
You should add a docs in the C API section for this.
And this kind of change definitely needs a ``NEWS`` entry.